src/js/components/modals/AppModalComponent.jsx

```diff
@@ -140,9 +141,9 @@ var AppModalComponent = React.createClass({
     });
 
     return (
-      <input type="submit"
-        className={classSet}
-        value={submitButtonTitle} />
+      <button type="submit" className={classSet}>
+        {submitButtonText}
+      </button>
```

Comment:
I think this `input` was a relic. Updated to `button`, since I was making changes anyway.